.github/workflows/s3-deploy.yml

```diff
@@ -0,0 +1,54 @@
+name: Deploy to S3
+
+on:
+  push:
+    branches: [ dev ]
+  pull_request:
+    branches: [ dev ]
```

Comment:
Do we have PRs upload to S3 currently? I think we only uploaded pushes to branches on the repo. Doing PRs might be a little dangerous.

Reply:
Yes, doing uploads for PRs is dangerous. Should only be for pushes.